lib/rules/prefer-numeric-literals.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * @fileoverview Rule to disallow `parseInt` in favor of binary, octal, and hexadecimal literals
+ * @author Annie Zhang, Henry Zhu
+ */
+
+"use strict";
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+
+module.exports = {
+    meta: {
+        docs: {
+            description: "disallow `parseInt` in favor of binary, octal, and hexadecimal literals",
```

Comment:
Sorry for all the nitpicking - caught one more thing! This should match the other descriptions and say `parseInt()`.